### PR TITLE
Create Dev Tool Function to write to SQS

### PR DESCRIPTION
**Pull request by praveenksamy148**
Upload file directly to aws s3 using node and send a queue to aws SQS

### CODE REVIEW

File: util/writeToSQS.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    credentials: creds
+});
+//upload file to s3 
+const s3 = new AWS.S3({ apiVersion: '2012-11-05' });
+const fileNo = Math.random();
+const uploadFile = (fileNo) => {
+    const params = {
+        Bucket: PRINTING_BUCKET_NAME,
+        Key: `folder/${fileNo}.pdf`,
+        Body: fs.readFileSync('./blank.pdf')
```
Comment (neelp03) [inline util/writeToSQS.js:23]: Why upload a pdf on local device?

File: util/writeToSQS.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    };
+    s3.upload(params, function (err) {
+        if (err) {
+            console.log(err);
+        }
+        else {
+            console.log('File uploaded successfully.');
```
Comment (neelp03) [inline util/writeToSQS.js]: instead of console logs, use the logger we have in Quasar/util/

File: util/writeToSQS.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    sqs.sendMessage(sqsParams, function (err) {
+        if (err)
+            console.log(err);
+        else
+            console.log('Successfull queue message sent');
+        console.log("URL: ", QueueUrl);
+        console.log("File Number: ", fileNo);
+
+        sqsParams.QueueUrl;
```
Comment (neelp03) [inline util/writeToSQS.js]: same here, use logger

File: util/writeToSQS.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    })
+}
+sendQueue(fileNo);
```
Comment (neelp03) [inline util/writeToSQS.js]: need EOF

File: util/writeToSQS.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+//send queue to sqs
+function sendQueue(fileNo) {
+    const sqs = new AWS.SQS({ apiVersion: '2012-11-05' });
+    const queueName = PRINTING_QUEUE_NAME;
+    const QueueUrl = `https://sqs.us-west-2.amazonaws.com/${ACCOUNT_ID}/${queueName}`
+    const sqsParams = {
+        MessageBody: JSON.stringify({
+            location: QueueUrl,
+            fileName: fileNo,
```
Comment (neelp03) [inline util/writeToSQS.js]:
```suggestion
            fileName: fileNo
```

**Review by neelp03 — CHANGES_REQUESTED** [review]
:d

**Comment by neelp03** [comment]
Also put writeToSQS in a new folder called tools

**Review by neelp03 — CHANGES_REQUESTED** [review]
for the functions you wrote can you add documentation in JS syntax please, that way if someone new were to come work on Core-v4, they would immediately understand what the function is doing without reading the body. Use these for reference:

[Inline JS Documentation Guide](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/)
[Documentation example in Core-v4](https://github.com/SCE-Development/Core-v4/blob/dev/api/peripheral_api/util/RFID-helpers.js#L10:L15)

**Review by neelp03 — APPROVED** [review]

File: util/tools/writeToSQS.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+      logger.info('Successfull queue message sent');
+      logger.info('URL: ', QueueUrl);
+      logger.info('File Number: ', fileNo);
+    }
+
+    sqsParams.QueueUrl;
+  });
+}
+
+sendQueue(fileNo);
```
Comment (evanugarte) [inline util/tools/writeToSQS.js]: can we just make a function at the bottom called main which does:
```js
function main() {
  const fileNo = Math.random();
  uploadFile(fileNo);
  sendQueue(fileNo);
}

main();
```

**Review by ShaivPandya — APPROVED** [review]

**Pull request merged**